pkg/sqlite/migrations/008_dependencies.go

```diff
@@ -0,0 +1,47 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+)
+
+const DependenciesMigrationKey = 8
+
+// Register this migration
+func init() {
+	registerMigration(DependenciesMigrationKey, dependenciesMigration)
+}
+
+var dependenciesMigration = &Migration{
+	Id: DependenciesMigrationKey,
+	Up: func(ctx context.Context, tx *sql.Tx) error {
+		sql := `
+		CREATE TABLE IF NOT EXISTS dependencies (
+			type TEXT,
+			package_name TEXT,
+			group_name TEXT,
+			version TEXT,
+			kind TEXT,
+			operatorbundle_name TEXT,
+			operatorbundle_version TEXT,
+			operatorbundle_path TEXT,
+			FOREIGN KEY(operatorbundle_name, operatorbundle_version, operatorbundle_path) REFERENCES operatorbundle(name, version, bundlepath) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED,
+			FOREIGN KEY(type, package_name, group_name, version, kind) REFERENCES dependencies(type, package_name, group_name, version, kind) ON DELETE CASCADE
+		);
+		`
+		_, err := tx.ExecContext(ctx, sql)
```

Comment:
this will need a data migration as well, to populate this table with data derived from the existing entries (i.e. api_requirer -> GVK dependency)

Reply:
Kevin and I talked about this yesterday and our initial thought was to do this later. However, given your input, it seems it is best just do it now. I'll address this.

Reply:
It's probably best to do it now so that we don't have to worry about a db in a "half migrated' state